crates/store/src/state.rs

```diff
@@ -226,7 +223,7 @@ impl State {
                 .map(|(note_index, note)| {
                     let details = match note {
                         OutputNote::Full(note) => Some(note.to_bytes()),
-                        OutputNote::Header(_) => None,
+                        OutputNote::Partial(_) | OutputNote::Header(_) => None,
```

Comment:
Similar to the comment I left in https://github.com/0xPolygonMiden/miden-node/pull/368#discussion_r1630144939 - getting a `Partial` note here should be an error. Let's create an issue for this.

Reply:
I've updated the code in order to return error in a case of invalid output note type. If I don't catch your point, please let me know, I will create an issue for this.